core/app/presenters/spree/variant_options_presenter.rb

```diff
@@ -0,0 +1,47 @@
+module Spree
+  class VariantOptionsPresenter
+    WORDS_CONNECTOR = ', '.freeze
+
+    attr_reader :variant
+
+    delegate :option_values, to: :variant
+
+    def initialize(variant)
+      @variant = variant
+    end
+
+    def to_sentence
+      options = option_values
```

Comment:
@KirillKayumov 
```suggestion
      options = option_values.include(:option_type).order('spree_option_types.position ASC')
```

Reply:
@damianlegawiec that was my initial approach but @piotrleniec-spark left a comment to not use preloading here, please take a look https://github.com/spree/spree/pull/9617#discussion_r340519146